time/unit.go

```diff
@@ -56,6 +56,14 @@ func (tu Unit) Value() (time.Duration, error) {
 	return 0, errUnrecognizedTimeUnit
 }
 
+// Count returns the number of units contained within the duration.
+func (tu Unit) Count(d time.Duration) (int, error) {
```

Comment:
`time.Duration` is aliased to int64, so dividing here leaves us with `int64`. No reason to loose range of values by down casting